iOS/TvShows/TvShows/Common/Network/Models/HttpStatusCode.swift

```diff
@@ -0,0 +1,19 @@
+enum HttpStatusCode: Int {
```

Comment:
What if some other status code occurs ? You'll get back request error ? 
Maybe it's easier to handle this in ranges and just handle the special cases.

Reply:
I agree. Will deal with it in the next PR where I have parsing error codes and messages for showing them to the user as alerts.